lib/lua/tableHelper.lua

```diff
@@ -400,6 +400,21 @@ function tableHelper.isEmpty(inputTable)
     return false
 end
 
+--- Retrieve a key associated with a specific value within a table.
+---
+--- @param inputTable table  The table to search within.
+--- @param searchValue any  The value to search for within the table.
+--- @return any|false  The key associated with the searchValue if found, or false if not found.
```

Comment:
The key cannot be any data type
```suggestion
--- @param searchValue string|number  The value to search for within the table.
--- @return string|number|false  The key associated with the searchValue if found, or false if not found.
```